presentconjugate.js

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
Could you please explain the purpose of this file. It look like it does the same functions as "present_conjugator.js"